tests/test_cmip6_datacube.py

```diff
@@ -36,9 +39,9 @@ def apply_attrs(xml: str):
         "huss_Amon_TaiESM1_historical_r1i1p1f1_gn_185001-201412.xml",
     ),
 )
-def test_datacube_helper(xml):
+def test_datacube_helper(xml, epsg4979_0_360_wkt):
     # Create item
-    item, _dc_ext = apply_attrs(xml)
+    item, _dc_ext = apply_attrs(xml, epsg4979_0_360_wkt)
```

Comment:
Is there a corresponding test file that could be used for the case where the proj is omitted (or defaults to EPSG:4326)?